README.md

```diff
@@ -1,9 +1,12 @@
 # illumos Project Discussion
 
-This repository is a preliminary holding area for a new shrink-to-fit process
-for collaboration on larger scale illumos projects.  This README itself will
-get fleshed out soon, but in the meantime there is a table of initial IPD
-(illumos project discussion) documents below:
+IPDs are a shrink-to-fit description of larger scale (in either code size or
+impact) project wishing to make changes to illumos.  They should describe the
+what and the why and give people the opportunity to comment on both.
+
+An IPD is at its heart, just a README.md in a numbered directory in this
```

Comment:
I guess I'm the only one who did an .adoc, haha. I'd say we should allow either. And if that's the case maybe rephrase this like "An IPD consists of either a markdown or asciidoc README file in a numbered directory".